### PR TITLE
adding filter statements to flatten the audit log

### PR DESCRIPTION
**Pull request by Makdaam**

### CODE REVIEW

File: pkg/genevalogging/genevalogging.go [hunk 1]
```diff
@@ -110,6 +110,27 @@ const (
...
+    Operation lift
+    Nested_under user
+    Add_prefix user_
+
+[FILTER]
+    Name nest
+    Match *
+    Operation lift
+    Nested_under responseStatus
+    Add_prefix response_
```
Comment (jim-minter) [inline pkg/genevalogging/genevalogging.go]: let's always match Nested_under and Add_prefix - so in this case `Add_prefix responseStatus_`

File: pkg/genevalogging/genevalogging.go [hunk 2]
```diff
@@ -110,6 +110,27 @@ const (
 	DB /var/lib/fluent/audit
 	Parser audit
 
+[FILTER]
```
Comment (jim-minter) [inline pkg/genevalogging/genevalogging.go]: checking k8s.io/apiserver/pkg/apis/audit/v1/types.go, I think we should also do impersonatedUser (if it exists).

Comment (Makdaam) [reply pkg/genevalogging/genevalogging.go]: added lift filter for impersonatedUser

File: pkg/genevalogging/genevalogging.go (hunk 1 above)
Comment (Makdaam) [reply pkg/genevalogging/genevalogging.go]: done

**Pull request merged**